protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -109,13 +111,11 @@ Note: this state represents any market that will be created, which currently mea
 
 ### Cancelled
 
-A market becomes Cancelled when a Market Proposal is successful and conditions are not met to transition the Market to the Active state during the Pending period, and one of the following apply:
-
-* the market reaches a timeout for the length of time in the Pending state; or
-* the Instrument hits expiry; or
-* governance votes (by approving a proposal to close the market) not to create the market after all.
+A market becomes Cancelled when a Market Proposal is successful and conditions are not met to transition the Market to the Active state during the Pending period, 
+and the trading terminated data source input rings, see [data sourcing](./0045-DSRC-data_sourcing.md).
+When a market transitions to a cancelled state all orders should be cancelled and collateral returned to respective parties general account for the relevant asset, all LP commitments should be cancelled and their bond returned to the general account for the relevant asset and any insurance pool balance should be transferred into the network treasury account for that asset. 
+All data sources that are only referenced by this market should be unregistered. 
 
-In future, it's expected that we will implement the functionality described in the white paper by which a proposer can revoke their proposal during the voting period by voting against it. In this case the status would also become Cancelled.
```

Comment:
I agree. 

I've removed it for now since with the growth of QA and them taking more ownership of specs (both good) it's harder to keep overall understanding of which bits of specs are to be implemented and which ones are really in *now* or to be done.